zeton/static/sass/sections/_welcome.scss

```diff
@@ -0,0 +1,44 @@
+.welcome {
+    .col {
+        .article__container {
+            border-radius: $zet-box-radius;
+            .article__item {
+                &:first-of-type {
+                    .article__image {
+
+                        left: 0;
+                        bottom: 0;
+                    }
+                }
+
+                &:last-of-type {
+
+                    .article__image {
+                        right: 0;
+                        top: 0;
+                    }
+                }
+            }
+        }
+    } 
+
+    @media only screen and (min-width: $zet-screen-lg-min) {
+        max-width: 1200px;
+        width: 100%;
+        margin: 0 auto;
+        display: flex;
+        justify-content: space-between;
+        align-items: stretch;
+
+        .col {
```

Comment:
you should never style bootstrap's classes, add a new one, your's own. What if we wanted to migrate from bootstrap to bulma?